migrate/20240111_core_count.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    alter_table(:vm_host) do
+      drop_constraint :core_allocation_limit
```

Comment:
This constraint helps to prevent over-allocation. What happens if different VMs select the same host simultaneously in the new scenario?

Reply:
ah yeah, we may need to materialize stuff to make a constraint work.